CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [7.1.3] - 2020-10-29
```

Comment:
Why not 7.2.0? This seems to be inconsistent with https://github.com/giantswarm/k8scloudconfig/pull/822

Reply:
Where would `release-v7.2.x` start?

Reply:
That's a good question. I don't know. `release-v7.2.x` should probably start with `v7.2.0` tag but that won't be the case with current automation. Maybe push it manually?